docs/_sources/api.rst.txt

```diff
@@ -0,0 +1,57 @@
+.. _jurity API:
+
+Jurity Public API
+====================
+
+.. automodule:: jurity
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Binary Classification Metrics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: jurity.classification.BinaryClassificationMetrics
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    
+Binary Fairness Metrics
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: jurity.fairness.BinaryFairnessMetrics
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Multiclass Fairness Metrics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: jurity.fairness.MultiClassFairnessMetrics
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    
+Binary Bias Mitigation
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: jurity.mitigation.BinaryMitigation
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Binary Recommenders Metrics
```

Comment:
Some of the doc string lengths go beyond 120 and are not being wrapped nicely.

Reply:
@dorukkilitcioglu some of the print statements are still not wrapped correctly... This *may* be an issue with me and doruk using a different sphinx version.

Reply:
Updated this now, the code changes didn't show up in the documentation, so I re-compiled and pushed.